app/lib/rocketchat/services/sdk.ts

```diff
@@ -158,6 +159,10 @@ class Sdk {
 	onStreamData(...args: any[]) {
 		return this.sdk.onStreamData(...args);
 	}
+
+	login(credentials: ICredentials) {
+		return this.sdk.login(credentials);
+	}
```

Comment:
Needed; even though is something being worked on the dehydration for `login` methods, without this `shareExtensionInit` might crash like on previous review